src/OpenLoco/Station.cpp

```diff
@@ -578,7 +578,11 @@ namespace OpenLoco
             {
                 if (cargo.quantity != 0 && cargo.origin != id())
                 {
-                    cargo.enroute_age = std::min(cargo.enroute_age + 1, 255);
+                    cargo.enrouteAge = std::min(cargo.enrouteAge + 1, 255);
+                }
+                else
+                {
+                    cargo.enrouteAge = 0;
```

Comment:
Take me through why this change? To me this looks like it forces all enroute cargo to have zero age

Reply:
Never mind i understand now you are just tidying it up